tests/mmm/test_budget_optimizer.py

```diff
@@ -28,38 +28,50 @@
             100,
             {"channel_1": (0, 50), "channel_2": (0, 50)},
             {
-                "channel_1": {
-                    "adstock_params": {"alpha": 0.5},
-                    "saturation_params": {"lam": 10, "beta": 0.5},
+                "saturation_params": {
```

Comment:
should we still test the scalar case?

Reply:
What do you mean here? What scalar case?